Release 3.1.5 doesn't work in Android

Seems like there are some regex patterns were added to release 3.1.5 which doesn't work in Android. Previous version (3.1.4) works well.
Here is stack trace:
```
Caused by java.util.regex.PatternSyntaxException: U_ILLEGAL_ARGUMENT_ERROR
^[\p{IsLetter}_][\p{IsLetter}\p{IsDigit}_]*
       at java.util.regex.Pattern.compileImpl(Pattern.java)
       at java.util.regex.Pattern.compile(Pattern.java:1344)
       at java.util.regex.Pattern.<init>(Pattern.java:1328)
       at java.util.regex.Pattern.compile(Pattern.java:950)
       at com.mitchellbosecke.pebble.lexer.LexerImpl.<clinit>(LexerImpl.java:2)
       at com.mitchellbosecke.pebble.PebbleEngine.getPebbleTemplate(PebbleEngine.java:11)
       at com.mitchellbosecke.pebble.PebbleEngine.lambda$getTemplate$2(PebbleEngine.java)
       at com.mitchellbosecke.pebble.PebbleEngine.lambda$getTemplate$2$PebbleEngine(PebbleEngine.java)
       at com.mitchellbosecke.pebble.-$$Lambda$PebbleEngine$RWC2jsSmCWn6HfuOZr1k4eoPeUE.apply(-.java:8)
       at j$.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:47)
       at j$.util.concurrent.ConcurrentMap$-EL.computeIfAbsent(ConcurrentMap.java:6)
       at com.mitchellbosecke.pebble.cache.template.ConcurrentMapTemplateCache.computeIfAbsent(ConcurrentMapTemplateCache.java:2)
       at com.mitchellbosecke.pebble.cache.template.ConcurrentMapTemplateCache.computeIfAbsent(ConcurrentMapTemplateCache.java)
       at com.mitchellbosecke.pebble.PebbleEngine.getTemplate(PebbleEngine.java:17)
       at com.mitchellbosecke.pebble.PebbleEngine.getTemplate(PebbleEngine.java:2)
```

According to this post [https://stackoverflow.com/questions/47922686/patternsyntaxexception-when-trying-to-validate-latin-characters](https://stackoverflow.com/questions/47922686/patternsyntaxexception-when-trying-to-validate-latin-characters) Android doesn't recognize `Is` aliases.

Still not working in Android:
```
Caused by: java.util.regex.PatternSyntaxException: U_ILLEGAL_ARGUMENT_ERROR
	^[\p{IsAlphabetic}_][\p{IsAlphabetic}\p{Digit}_]*
	at java.util.regex.Pattern.compileImpl(Native Method)
	at java.util.regex.Pattern.compile(Pattern.java:1340)
	at java.util.regex.Pattern.<init>(Pattern.java:1324)
	at java.util.regex.Pattern.compile(Pattern.java:946)
	at com.mitchellbosecke.pebble.lexer.LexerImpl.<clinit>(LexerImpl.java:93)
```

--- a/pebble/src/main/java/com/mitchellbosecke/pebble/lexer/LexerImpl.java
+++ b/pebble/src/main/java/com/mitchellbosecke/pebble/lexer/LexerImpl.java
@@ -92,7 +92,7 @@ public final class LexerImpl implements Lexer {
   private static final Pattern REGEX_IDENTIFIER;
   
   static {
-    if ("The Android Project".equals(System.getProperty("java.vendor")) {
+    if ("The Android Project".equals(System.getProperty("java.vendor"))) {
       REGEX_IDENTIFIER = Pattern.compile("^[\\p{Letter}_][\\p{Letter}\\p{Digit}_]*");
     } else {
       // Standard

--- a/pebble/src/main/java/com/mitchellbosecke/pebble/lexer/LexerImpl.java
+++ b/pebble/src/main/java/com/mitchellbosecke/pebble/lexer/LexerImpl.java
@@ -89,8 +89,16 @@ public final class LexerImpl implements Lexer {
   /**
    * Static regular expressions for identifiers.
    */
-  private static final Pattern REGEX_IDENTIFIER = Pattern
-      .compile("^[\\p{IsAlphabetic}_][\\p{IsAlphabetic}\\p{Digit}_]*");
+  private static final Pattern REGEX_IDENTIFIER;
+  
+  static {
+    if ("The Android Project".equals(System.getProperty("java.vendor")) {
+      REGEX_IDENTIFIER = Pattern.compile("^[\\p{Letter}_][\\p{Letter}\\p{Digit}_]*");
+    } else {
+      // Standard
+      REGEX_IDENTIFIER = Pattern.compile("^[\\p{IsLetter}_][\\p{IsLetter}\\p{IsDigit}_]*");
+    }
+  }
 
   private static final Pattern REGEX_LONG = Pattern.compile("^[0-9]+L");
 

